guides/get-started/transactions.md

```diff
@@ -2,21 +2,42 @@
 title: Send and Receive Money
 ---
 
-Now that you have an account, you can send and receive funds through the Stellar network. If you haven’t created an account yet, read [step 2 of the Get Started guide](./create-account.md).
+_Before continuing with the following examples for Stellar in code, consider going through the
```

Comment:
This paragraph is the only added section to this file — everything else is reformatting long lines.